shared/controller-api/src/main/swagger/Controller.yaml

```diff
@@ -26,28 +26,33 @@ info:
     url: "http://www.apache.org/licenses/LICENSE-2.0"
 basePath: "/v1"
 tags:
-- name: "Scopes"
-  description: "Scope related APIs"
-  externalDocs:
-    description: "Find out more about pravega"
-    url: "https://pravega.io"
-- name: "Streams"
-  description: "Stream related APIs"
-  externalDocs:
-    description: "Find out more about pravega"
-    url: "https://pravega.io"
-- name: "ReaderGroups"
-  description: "Reader group related APIs"
-  externalDocs:
-    description: "Find out more about pravega"
-    url: "https://pravega.io"
+  - name: "Scopes"
```

Comment:
Is it possible to revert these spacing changes?

Reply:
I didn't actually intend to make these changes, was some automatic linting from IntelliJ. However, it does look like these changes makes the indentation consistent across the entire file. For example in certain places it indents starting on the first non '-' and others it indents starting at the '-'. Before the PR, it seems majority followed the latter convention.